### PR TITLE
Add `accept` attribute to the video upload

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description
Add the `accept` attribute to the video upload. Adding `video/*` alone isn't enough because safari will ignore MP4. The other change is a safe fallback for `PREFERRED_EMAIL_NAME`

ref: https://stackoverflow.com/questions/19107685/safari-input-type-file-accept-video-ignores-mp4-files
## Related Tickets & Documents
n/a
## Added tests?

- [x] no, because they aren't needed

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

File: app/controllers/email_subscriptions_controller.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def unsubscribe
     if verified_params[:expires_at] > Time.current
       user = User.find(verified_params[:user_id])
       user.update(verified_params[:email_type] => false)
-      @email_type = PREFERRED_EMAIL_NAME[verified_params[:email_type]]
+      @email_type = PREFERRED_EMAIL_NAME.fetch(verified_params[:email_type], "this list")
```
Comment (benhalpern) [inline app/controllers/email_subscriptions_controller.rb]: Unrelated change?

If this change is intended and works I think it's fine to keep in.

Comment (maestromac) [reply app/controllers/email_subscriptions_controller.rb]: Yep intentional, suggested by https://github.com/thepracticaldev/dev.to/pull/6027#discussion_r378629230

Comment (citizen428) [reply app/controllers/email_subscriptions_controller.rb]: I see @benhalpern's point, it's confusing to have in this PR as it's not related per se, but I think we can leave it in.
